generated/nidaqmx/base_interpreter.py

```diff
@@ -0,0 +1,1630 @@
+# Do not edit this file; it was automatically generated.
+from nidaqmx.constants import (
+    ACExcitWireMode, AccelChargeSensitivityUnits, AccelSensitivityUnits,
+    AccelUnits, AcquisitionType, AngleUnits, AngularVelocityUnits,
+    BridgeConfiguration, BridgeElectricalUnits, BridgePhysicalUnits,
+    BridgeUnits, CJCSource, ChargeUnits, CountDirection,
+    CounterFrequencyMethod, CurrentShuntResistorLocation, CurrentUnits,
+    DigitalPatternCondition, EddyCurrentProxProbeSensitivityUnits, Edge,
+    EncoderType, EncoderZIndexPhase, ExcitationSource,
+    ForceIEPESensorSensitivityUnits, ForceUnits, FrequencyUnits, FuncGenType,
+    GpsSignalType, LVDTSensitivityUnits, LengthUnits, Level, LineGrouping,
+    Polarity, PressureUnits, RTDType, RVDTSensitivityUnits,
+    ResistanceConfiguration, ResistanceUnits, Signal, SignalModifiers, Slope,
+    SoundPressureUnits, StrainGageBridgeType, StrainGageRosetteType,
+    StrainUnits, TEDSUnits, TemperatureUnits, TerminalConfiguration,
+    ThermocoupleType, TimeUnits, Timescale, TorqueUnits, UnitsPreScaled,
+    VelocityIEPESensorSensitivityUnits, VelocityUnits, VoltageUnits,
+    WindowTriggerCondition1, WriteBasicTEDSOptions)
+
+class BaseInterpreter:
+    """
+    Contains signature of functions for all DAQmx APIs.
+    """
+    def add_ai_accel_4_wire_dc_voltage_chan(
+            self, physical_channel, name_to_assign_to_channel="",
+            terminal_config=TerminalConfiguration.DEFAULT, min_val=-5.0,
+            max_val=5.0, units=AccelUnits.G, sensitivity=1000.0,
+            sensitivity_units=AccelSensitivityUnits.MILLIVOLTS_PER_G,
+            voltage_excit_source=ExcitationSource.INTERNAL,
+            voltage_excit_val=0.0, use_excit_for_scaling=False,
+            custom_scale_name=""):
```

Comment:
Consider generating type annotations.

Reply:
Ok, if we don't have the needed metadata, then let's not add type annotations now.

I think type annotations should be added at some point, but it doesn't have to be part of this project. https://github.com/ni/nidaqmx-python/issues/209